### DISCUSSION TITLE
Switch source from playlist to input.http and vice versa

### DISCUSSION THREAD

**Discussion by caf2022** [savonet/liquidsoap#2389] [User Support]
How to I can Switch the source input between playlist/input.http ?

In telnet I have a option to define the input.http url, I need to know how to I can Switch from playlist to this input.http that I was defined on xxx.url and xxx.start and then when xxx.stop back to the playlist.

Someone can help me with this please?

I'm using the v2.0.4

**Comment by caf2022** [comment]
Now I understood that the fallback that do this Switch, correct?

**Comment by toots** [comment]
Hi,

`switch` are operators that can be programmed to switch to a source whenever appropriate. `fallback` is a simpler version of `switch` that selects the first source available.

I think that you could check out the liquidsoap book here for more context and examples: https://www.liquidsoap.info/doc-dev/book.html
